spec/dummy/app/avo/actions/test/close_modal.rb

```diff
@@ -9,5 +9,10 @@ def handle(**args)
     TestBuddy.hi "Hello from Avo::Actions::Test::CloseModal handle method"
     succeed "Modal closed!!"
     close_modal
+
+    # Test custom added turbo_streams
+    enhance_response turbo_stream: -> {
+      turbo_stream.set_title("Dummy action custom stream ;)")
+    }
   end
```

Comment:
Let's rename it to `respond_with` and pass in an array or a single `turbo_stream` action.

The reason why I'm thinking to not do this `turbo_stream: ->` si because we only respond with turbo streams... It's a bit redundant to declare it every time, right?
Do you see any caveats?

```suggestion
    enhance_response turbo_stream: -> {
      turbo_stream.set_title("Dummy action custom stream ;)")
    }
  end
```

Reply:
> Let's rename it to `respond_with` ...

`respond_with` kinda sounds like is overriding the default responses.

> ...and pass in an array or a single `turbo_stream` action.
The reason why I'm thinking to not do this `turbo_stream: ->` si because we only respond with turbo streams... It's a bit redundant to declare it every time, right?
Do you see any caveats?

You mean something like (I'll use `enhance_response` until renaming):
```ruby
enhance_response turbo_stream: turbo_stream.set_title("Dummy action custom stream ;)")

# OR

enhance_response turbo_stream.set_title("Dummy action custom stream ;)")
```

And for array:

```ruby
enhance_response turbo_stream: [
  turbo_stream.set_title("Dummy action custom stream ;)"),
  turbo_stream...
]

# OR

enhance_response [
  turbo_stream.set_title("Dummy action custom stream ;)"),
  turbo_stream...
]
```

> Do you see any caveats?

None if `turbo_stream` is present.